src/azure-cli-core/azure/cli/core/authentication.py

```diff
@@ -35,16 +35,21 @@ def __init__(self, credential, **kwargs):
         # _external_credentials and _resource are only needed in Track1 SDK
         self._external_credentials = kwargs.pop("external_credentials", None)
         self._resource = kwargs.pop("resource", None)
+        self._scopes = kwargs.pop("scopes", None)
```

Comment:
>self._scopes = kwargs.pop("scopes", None) [](start = 8, length = 41)

seems not needed since `scopes` should be set in `get_token`

Reply:
See my detailed response at the related discussion.